### PR TITLE
feat(editor): Support autologin for upgrade path

### PR DESCRIPTION
**Pull request by RicardoE105**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/src/stores/ui.store.ts [hunk 1]
```diff
@@ -329,18 +328,29 @@ export const useUIStore = defineStore(STORES.UI, {
 			};
 		},
 		upgradeLinkUrl() {
-			return (source: string, utm_campaign: string): string => {
-				const linkUrlTranslationKey = this.contextBasedTranslationKeys
```
Comment (mutdmour) [inline packages/editor-ui/src/stores/ui.store.ts:333]: can we clean this up then if we are not using them?

Comment (RicardoE105) [reply packages/editor-ui/src/stores/ui.store.ts:333]: We are still using contextBasedTranslationKeys in a bunch of places!

Comment (mutdmour) [reply packages/editor-ui/src/stores/ui.store.ts:333]: I meant specifically `.upgradeLinkUrl` there

Comment (RicardoE105) [reply packages/editor-ui/src/stores/ui.store.ts:333]: ahhhh gotcha!

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 2]
```diff
@@ -1924,9 +1924,6 @@
 	"contextual.users.settings.unavailable.button.desktop": "View plans",
 	"contextual.communityNodes.unavailable.description.desktop": "Community nodes feature is unavailable on desktop. Please choose one of our available self-hosting plans.",
 	"contextual.communityNodes.unavailable.button.desktop": "View plans",
-	"contextual.upgradeLinkUrl": "https://n8n.io/pricing/",
-	"contextual.upgradeLinkUrl.cloud": "https://app.n8n.cloud/account/change-plan",
-	"contextual.upgradeLinkUrl.desktop": "https://n8n.io/pricing/?utm_source=n8n-internal&utm_medium=desktop",
```
Comment (mutdmour) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:1929]: this is still used in SettingsCommunityNodesView.. maybe we can just delete the desktop reference there

Comment (RicardoE105) [reply packages/editor-ui/src/plugins/i18n/locales/en.json:1929]: wth I did a text search

Comment (RicardoE105) [reply packages/editor-ui/src/plugins/i18n/locales/en.json:1929]: All should use the same function in the UI store to redirect to the upgrade path. So will update to use the function and we can get rid of the context.upgradeLinkUrl.*

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.11.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.11.0)

**Pull request merged**